### PR TITLE
Add links to footer

### PR DESCRIPTION
**Pull request by PapaCapricorn**
My first pull request...be gentle! Thought I'd try to ape this together for fun. Just guessing wildly at the proper formatting. Hope it saves you a bit of time to have this started anyway.

Expanded links in footer to include General Contact | Support | List Your Events | Business Inquiries

image

### CODE REVIEW

**Comment by jtrussell** [comment]
Awesome, thanks @PapaCapricorn! There are two more places to update to round this one out if you're up for it :D.

- https://github.com/mythgard-hub/mythgard-hub/blob/master/next/next.config.js This exposes env vars to the Next app.
- https://github.com/mythgard-hub/mythgard-hub/blob/master/kube-config/manifest.yml This is what we're using to add env vars to the production environment.

You should be able to follow your nose if you look at the treatment the other email vars got.

**Comment by PapaCapricorn** [comment]
Ok thanks @jtrussell - I followed my nose and it led me here. What is that smell?

'next.config.js' and 'manifest.yml' have been updated

**Comment by jtrussell** [comment]
Nice! The manifest file just needed an update for both container specs (I added the second one).

File: next/components/footer.js [hunk 1]
```diff
@@ -31,8 +31,11 @@ export default function Footer() {
         }
       `}</style>
       <span>&copy; {new Date().getFullYear()} Mythgard</span>
-      <a href={`mailto:${process.env.EMAIL_MG_SUPPORT}`}>Email</a>
       <a href={process.env.DISCORD_INVITE_URL}>Discord</a>
+      <a href={`mailto:${process.env.EMAIL_MG_CONTACT}`}>General Contact</a>
+      <a href={`mailto:${process.env.EMAIL_MG_SUPPORT}`}>Support</a>
+      <a href={`mailto:${process.env.EMAIL_MG_EVENTS}`}>List Your Event</a>
+x      <a href={`mailto:${process.env.EMAIL_MG_BUSINESS}`}>Business Inquiries</a>
```
Comment (evsheffield) [inline next/components/footer.js]: Looks like this little "x" snuck in here. ;)

Comment (PapaCapricorn) [reply next/components/footer.js]: Thanks @evsheffield - good catch! (Those pesky errant keystrokes mumble mumble...)

Should be fixed now.

**Pull request merged**